docs/source/cli/model-training-inference/output-remapping.rst

```diff
@@ -1,6 +1,220 @@
+By default, the output prediction results after ``gconstruct.remap_result``
+are stored in parquet format. The prediction results for different node
+types are stored in separate directories, each named after the
+corresponding node type. The prediction results for different edge
+types are stored in separate directories, each named after the
+corresponding edge type.
+
+The content of the node prediction results will look like following:
+
+.. code-block:: bash
+
+    predict_dir/
+        ntype0:
+            predict-00000_00000.parquet
+            predict-00000_00001.parquet
+            ...
+        ntype1:
+            predict-00000_00000.parquet
+            predict-00000_00001.parquet
+            ...
+
+The content of the edge prediction results will look like following:
+
+.. code-block:: bash
+
+    predict_dir/
+        etype0:
+            predict-00000_00000.parquet
+            predict-00000_00001.parquet
+            ...
+        etype1:
+            predict-00000_00000.parquet
+            predict-00000_00001.parquet
+            ...
+
+For multi-task learning tasks, there can be multiple prediction results
+for different tasks. (Details can be found in :ref:`Multi-task
+Learning Output<multi-task-learning-output>`). The task specific
+prediction results are also processed by the ``gconstruct.remap_result`` module.
+The content of the output will look like following:
+
+.. code-block:: bash
+
+    prediction_dir/
+        edge_regression-paper_cite_paper-year/
+            paper_cite_paper/
+                predict-00000_00000.parquet
+                predict-00000_00001.parquet
+                ...
+        node_classification-paper-venue/
+            paper/
+                predict-00000_00000.parquet
+                predict-00000_00001.parquet
+        ...
+
+The content of a node prediction result file will look like following:
+
++-----+----------------+
+| nid |   pred         |
++=====+================+
+| n0  | 0.2964, 0.7036 |
++-----+----------------+
+| n1  | 0.1862, 0.8138 |
++-----+----------------+
+| n10 | 0.9778, 0.0222 |
++-----+----------------+
+
+.. note::
+
+    ``gconstruct.remap_result`` uses `nid` as the default column name
+    for node IDs and `pred` as the default column name for prediction results.
+
+The content of an edge prediction result file will look like following:
+
++---------+---------+----------------+
+| src_nid | dst_nid |      pred      |
++=========+=========+================+
+|    n0   |   n32   | 0.2964, 0.7036 |
++---------+---------+----------------+
+|    n1   |   n21   | 0.1862, 0.8138 |
++---------+---------+----------------+
+|    n10  |   n2    | 0.9778, 0.0222 |
++---------+---------+----------------+
+
+.. note::
+
+    ``gconstruct.remap_result`` uses `src_nid` as the default column name
+    for source node IDs, `dst_nid` as the default column name for
+    destination node IDs and `pred` as the default column name for prediction results.
+
+Run remap_result Command
```

Comment:
Suggest to move this `Run remap_result Command` section ahead before the `Output node embeddings` section. This could be a more nature order of narrative.

Reply:
But Output node embedding is more important.
If users are not going to use remap_reuslt, they can skip the last part.